src/App.tsx

```diff
@@ -1,73 +1,45 @@
-import React from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
-
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
 
-  return (
-    <div className="container">
-      <main className="section is-flex is-flex-direction-column">
-        <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
-        </h1>
+import React, { useMemo, useState } from 'react';
```

Comment:
Consider checking if `person.slug` or any other identifier used for keys in list rendering is unique and stable to avoid potential issues.